lib/sparql_to_sw_solr.rb

```diff
@@ -2,5 +2,6 @@ module SparqlToSwSolr
   require_relative '../config/ckey_blacklist'
   require_relative 'sparql_to_sw_solr/instance_solr_doc'
   require_relative 'sparql_to_sw_solr/instance_title_fields'
+  require_relative 'sparql_to_sw_solr/language_field'
```

Comment:
Actually don't need to include it at this level (makes for extra maintenance), because it's already included by `instance_solr_doc` and that's the only thing that needs it.